tools/build/third_party/perfetto/perfetto.BUILD

```diff
@@ -218,8 +224,14 @@ cc_library(
 )
 
 cc_library(
-    name = "tracing_core",
+    name = "tracing_ipc",
```

Comment:
Why did these targets get merged?

Reply:
There are some cross-dependencies between the two targets (once you add the rest of the missing files in) I could find no other way to make all of the builds happy.